packaging/bin/pki-enroll-pkcs12.sh

```diff
@@ -9,6 +9,7 @@ enroll() {
 	local ovirt_san="$6"
 	local keep_key="$7"
 	local ca_file="$8"
+        local days="$9"
```

Comment:
indentation throughout the file

Reply:
Oops, still tabs in a source file. 🙄
This should be fixed in a followup patch.